jest.config.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
+const baseConfig = require('./jest.base-config');
+
 module.exports = {
-  name: 'Unit test',
-  testMatch: ['<rootDir>/packages/**/__tests__/?(*.)+(spec|test).js'],
-  modulePathIgnorePatterns: ['.cache'],
-  transform: {},
-  setupFilesAfterEnv: ['<rootDir>/test/unit.setup.js'],
+  ...baseConfig,
+  projects: [
+    '<rootDir>/packages/core/*/jest.config.js',
```

Comment:
Can we add the other workspaces (providers / generators ...) in advance ?

Reply:
do we really need the /jest.config.js at the end ? isn't it the default ?

Reply:
I wanted to avoid passing it as well at first but it seems we do need it, otherwise, it will try to run tests in all targeted folders (even when the `jest.config.js` is not present) using default jest configuration values, thus generating a lot of issues.